WebApplicationTests/WebApplicationWeb/PageObjects/SearchResultsPage.cs

```diff
@@ -0,0 +1,9 @@
+using OpenQA.Selenium;
+
+namespace WebApplicationWeb;
+
+public class SearchResultsPage
+{
+    private IWebDriver _driver;
```

Comment:
You should use BrowserFactory class to use FindElements method instead of creating new instance of IWebDriver.